src/libsync/clientsideencryption.cpp

```diff
@@ -956,6 +956,7 @@ std::string FolderMetadata::decryptJsonObject(const std::string& encryptedMetada
     qCInfo(lcCse()) << "currently decrypted" << std::string( (char*) out, outlen);
     qCInfo(lcCse()) << "Current decrypt  length" << outlen;
 
+// TODO: figure out why this is not working.
```

Comment:
I know why it bis because we don't properly set the tag I think.

I'll talk to you tomorrow (or fix it tomorrow as it is easy for me as I know how it works).